lib/Git/ProcessBuilder.php

```diff
@@ -27,7 +27,7 @@ public function __construct(array $arguments = array())
     {
         array_unshift($arguments, 'git');
         $gitDir = \Mesamatrix::path(\Mesamatrix::$config->getValue("info", "private_dir"))."/";
-        $gitDir .= \Mesamatrix::$config->getValue("git", "dir");
+        $gitDir .= \Mesamatrix::$config->getValue("git", "dir", "mesa.git");
```

Comment:
Why did you add a default value to all the getValue()? Is it a restriction from the logger that I don't see or is it because you prefer that way?